sources/packages/backend/apps/db-migrations/src/sql/SFASIndividuals/Add-student-id-sfas-individuals.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE
+    sims.sfas_individuals
+ADD
+    COLUMN IF NOT EXISTS student_id INTEGER REFERENCES sims.students(id);
+
+COMMENT ON COLUMN sims.sfas_individuals.student_id IS 'Student id used to import data to sims.disbursement_overawards table.';
```

Comment:
The column can state what it is instead of the purpose because it was added.
Sugestion
```
Existing student id identified by the combination of SIN, last name, and date of birth.
```